### DISCUSSION TITLE
Json validation doesn`t work properly

### DISCUSSION THREAD

**Discussion by mirket999** [Kong/insomnia#6550] [Help]
I have just switched to Insomnia from Postman. 
I tryna do Json schema validation but it is not so effective for me.
First I tried to use my existing method in Postman:

```
const response = await insomnia.send();
const jsonResponse = JSON.parse(response.data);
const expectedSchema= {...};

//jsonResponse.to.have.jsonSchema(expectedSchema)
//expect(jsonResponse).to.be.jsonSchema(expectedSchema);
//assert.jsonSchema(jsonResponse, expectedSchema);
//jsonResponse.should.be.jsonSchema(expectedSchema);
//chai.tv4.addSchema(jsonResponse, expectedSchema);

//var valid = chai.tv4.validate(jsonResponse, expectedSchema);
```
 
Even though Insomnia notifies that Chai library is used for assertions, I couldn`t achieve to work with them.

Then I used a method for validation which iterates over schema and verifies the actual response body but it doesn`t work well for complicated schemas. 
How do you validate your schema and is it needed to have additional configurations to make it run?

**Comment by kobenguyent** [comment]
Hope this https://github.com/Kong/insomnia/pull/6532 to be considered by the team and solve the issue.

**Comment by subnetmarco** [comment]
Working on this as we speak, duplicate of https://github.com/Kong/insomnia/discussions/6268 so closing here. Indeed we are looking at #6532 as the resolution.
